identifyyeastcells.py

```diff
@@ -0,0 +1,1146 @@
+            each individual image. In some specific cases it is better to use manually precomputed background loaded from file.</li>
+            </ul>"""%globals())
+        
+        self.average_cell_diameter = cps.Float(
+            "Average cell diameter in pixels",
+            30.0, minval=10, doc ='''\
+            The average cell diameter is used to scale many algorithm parameters. 
+            Please use e.g. ImageJ to measure the average cell size in pixels.
+            '''%globals()
+            )
+
+        self.advanced_cell_filtering = cps.Binary(
+            'Do you want to filter cells by area?', False, doc="""
+            This parameter is used for cell filtering. The algorithm creates many more cell candidates than finally accepted cells (these
+            cells overlap with each other). At the final phase of algorithm the cells are selected from the ensamble of cell candidates based on the 
+            "quality" measure. Use this option if you want to absolutely prohibit too small (or too big) cells to be chosen (regardless of "quality"). 
+            """%globals())
+
+        self.min_cell_area = cps.Integer(
+            "Minimal area of accepted cell in pixels",
+            900, minval=10, doc ='''\
+            <i>(Used only when you want to filter cells based on area)</i><br>
+            The minimum cell area is used while final filtering of cells. 
+            Please use e.g. ImageJ to measure the average cell size in pixels.
+            '''%globals()
+            )
+
+        self.max_cell_area = cps.Integer(
+            "Maximum area of accepted cell in pixels",
+            5*900, minval=10, doc ='''\
+            <i>(Used only when you want to filter cells based on area)</i><br>
+            The maximum cell area is used while final filtering of cells. 
+            Please use e.g. ImageJ to measure the average cell size in pixels.
+            '''%globals()
+            )
+
+        
+        self.background_brighter_then_cell_inside = cps.Binary(
+            'Is the area without cells (background) brighter then cell interiors?', True, doc="""
+            Please check if the area inside of the cells is <b>darker</b> then the area without the cells (background). Use e.g. ImageJ to measure 
+            average intensity.
+            """%globals()
+            )
+
+        self.bright_field_image = cps.Binary(
+            'Do you want to segment brightfield images?', True, doc="""
+            Choose this option if you want to segment a brightfiled image. For segmentation of fluorescent images please answer "No". 
+            """%globals()
+            )
+
+        self.advanced_parameters = cps.Binary(
+            'Use advanced configuration parameters', False, doc="""
+            Do you want to use advanced parameters to configure plugin? They allow for more flexibility,
+            however you need to know what you are doing.
```

Comment:
Perhaps:

> They allow for more flexibility, however you need to know what you are doing.

They allow for more flexibility, and require understanding of the algorithm to configure well.